client/dist/style.css

```diff
@@ -229,57 +229,6 @@ body, button {
 
 /********************************************/
 
-.star {
```

Comment:
Were these star styles accidentally removed?

Reply:
@boyiq  I didn't make any changes after merging, but that def looks incorrect so something might have gone wrong with my merge. Let's look together when we hop on in a bit